src/components/HeatMap.jsx

```diff
@@ -41,7 +40,7 @@ const HeatmapChart = ({ allWeather, currentDate }) => {
           if (
             tickDate.toDateString() === new Date(currentDate).toDateString()
           ) {
-            return format(tickDate, "HH");
+            return getHour(tickDate);
```

Comment:
When I run the branch locally I get a getHour not defined error.

Reply:
I have updated the function to tickDate.getHours() and it works for me locally.

Reply:
Ok, thanks!

An import from utils/date.js would fix it too.

The merge conflict caused the error, I suppose.